# TODO: Add description

## Rule: `erb-rubocop`

TODO: Add description

## Examples

### ❌ Incorrect

```erb
<!-- TODO: Add incorrect example -->
```

### ✅ Correct

```erb
<!-- TODO: Add correct example -->
```

## Configuration

This rule has no configuration options.
